pandas/tests/groupby/test_groupby.py

```diff
@@ -2763,13 +2764,23 @@ def test_rolling_wrong_param_min_period():
         test_df.groupby("name")["val"].rolling(window=2, min_period=1).sum()
 
 
-def test_by_column_values_with_same_starting_value():
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        object,
+        pytest.param(
+            "string[pyarrow_numpy]",
+            marks=pytest.mark.skipif(pa_version_under7p0, reason="arrow not installed"),
```

Comment:
Could you use an `importorskip` in the test when `dtype` is `pyarrow_numpy`? (avoids having to change `pa_version_under7p0` every time we bump pyarrow)

Reply:
thoughts about creating a pyarrow_installed variable?

I like this pattern a bit better than the if inside the test

Reply:
> thoughts about creating a pyarrow_installed variable?

Sure this would be good too

Reply:
done